src/main/webapp/src/app/recommendation-form/recommendation-form.component.html

```diff
@@ -0,0 +1,74 @@
+<h1>Hidden Gems Recommendations</h1>
+<h4>Get a restaurant recommendation tailored just for you!</h4>
+
+<form [formGroup]="preferenceForm" (ngSubmit)="onSubmit()">
+  <h2>Your Restaurant Preferences</h2>
+  <p>Please select your restaurant preferences in the form below. If you have no preference for a field, please feel free to leave it blank</p>
```

Comment:
set column to 100 , so it will transfer excessive part of code to the next line

Reply:
technically it's only java that has a width limit in it's style guide, that said if you can break up lines so you're not scrolling infinitely it can be useful, up to you though.

Reply:
Done